models/issue_comment.go

```diff
@@ -120,6 +120,9 @@ type Comment struct {
 	AssigneeID       int64
 	RemovedAssignee  bool
 	Assignee         *User `xorm:"-"`
+	ResolveDoerID    int64
+	IsResolved       bool  `xorm:"-"`
```

Comment:
Can you confert this to a function?

`func (c *Comment) IsResolved() {...}`

Reply:
hint: do it in a single xorm query

Reply:
I don't think  it's necessary to be as an function, because it will use in  many places on tmpl file. and the results is same.so I think this way is more simple

Reply:
This can cause bugs like: #11033 or #10968

Reply:
I think `IsResolved()` should be a function too.

Reply:
this can be "resolved"